src/Apps/Artist/Routes/Overview/state.ts

```diff
@@ -40,6 +41,48 @@ export const initialState = {
   showZeroState: false,
 }
 
+// Returns a string representing the query part of a URL.
+// It removes default values.
+export const urlFragmentFromState = (state: State, extra?: Partial<State>) => {
+  const filters = Object.entries(state).reduce((acc, [key, value]) => {
+    if (isDefaultFilter(key, value)) {
+      return acc
+    } else {
+      return { ...acc, [key]: value }
+    }
+  }, {})
+
+  return qs.stringify({
+    ...filters,
+    ...extra,
+  })
+}
+
+// This is used to remove default state params that clutter up URLs.
+const isDefaultFilter = (filter, value): boolean => {
+  if (filter === "major_periods" || filter === "attribution_class") {
+    return value.length === 0
+  }
+
+  if (filter === "sort") {
+    return value === "-decayed_merch"
+  }
+
+  if (filter === "price_range" || filter === "height" || filter === "width") {
```

Comment:
For code like this, where we're checking to see if one variable (`filter`) is one of a few options, I'd recommend something like this:

```js 
if (["price_range", "height", "width"].includes(filter)) {
```

But that also breaks parallelism with the other `if` statements, hmm... 

I'm fine with this as-is. A `switch` might be an idiomatic solution, but I'll leave it up to you 👍

Reply:
I agree that there are probably ways to cut down the number of lines of code in this function, but I'm not personally offended by its current state. I find it pretty readable, and I think trying to remove the `if` statements would result in some trickery that required more thinking to understand. I'm planning to leave this as-is.